lib/experimental/block-bindings/sources/pattern.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Add the metadata source to the block bindings API.
+ *
+ * @package gutenberg
+ */
+
+if ( function_exists( 'register_block_bindings_source' ) ) {
+	$pattern_source_callback = function ( $source_attrs, $block_content, $block, $block_instance ) {
+		if ( ! _wp_array_get( $block_instance->attributes, array( 'metadata', 'id' ), false ) ) {
+			return;
+		}
+		$block_id = $block_instance->attributes['metadata']['id'];
+		return _wp_array_get( $block_instance->context, array( 'pattern/overrides', $block_id ), false );
```

Comment:
Same here. We should return `null` if there's no overrides for this block or else the frontend will just render an empty content.

```suggestion
		return _wp_array_get( $block_instance->context, array( 'pattern/overrides', $block_id ), null );
```

Reply:
Done, thanks! 🙌